pkg/exchange/ftx/exchange.go

```diff
@@ -240,9 +240,20 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 		if since.IsZero() || current.Unix() == since.Unix() {
 			break
 		}
+
+		if options.Limit != 0 && options.Limit <= len(lines) {
```

Comment:
use `<` ? limit = fetched kline length means we might have more after this time point?

Reply:
yeh, it might be more than we need here, but we will handle the limit part later in line 283